src/main/java/org/jabref/gui/collab/experimental/ExternalChangeDetailsViewFactory.java

```diff
@@ -0,0 +1,8 @@
+package org.jabref.gui.collab.experimental;
+
+public class ExternalChangeDetailsViewFactory {
+
+    public ExternalChangeDetailsView create(ExternalChangeResolver resolver) {
+        return null;
```

Comment:
Is this intended?

Reply:
It is still experimental. I'm redesigning the entire external changes resolver component in the experimental package. It currently only has the class skeletons.